### DISCUSSION TITLE
Where i can learn full simple documentation to avr-hal? please

### DISCUSSION THREAD

**Discussion by IhorLihvan** [Rahix/avr-hal#574] [Q&A]
Where i can learn full simple documentation to avr-hal? please

**Comment by Rahix** [comment]
Unfortunately, there isn't too much documentation about `avr-hal` yet.  Maybe this video by Low Level Learning can help you get started: https://www.youtube.com/watch?v=ZPSqhb4KKNc

From the project itself, you'll have to use the crate documentation and examples and figure things out.  If questions arise, feel free to ask, of course.
